docs/04-resources/08-runes/airdrop/airdrop-setup.md

```diff
@@ -0,0 +1,128 @@
+---
+sidebar_position: 2
+title:  Setup
+sidebar_label: Setup
```

Comment:
Below you suggested `Deploy` to be Airdrop Machine` and this `Airdrop Engine`
I used Just `Setup` and `Deploy` because its duplicate with the top title, and i only changed the sidebar title which will not affect SEO

Reply:
Ok. Cool

Just match the side bar label with the title, setup only is not descriptive

Reply:
Ok